lib/generator.js

```diff
@@ -98,7 +98,7 @@
         var self = this;
         
         self._options = options || {};
-        self._config = options.config || {};
+        self._config = self._options.config || {};
         
         _logger.debug("Launching with config:\n%s", JSON.stringify(self._config, null, "  "));
         if (options.photoshopVersion) {
```

Comment:
@iwehrman I've updated it. Squashed my commits so your diff became outdated. (sorry)